static/scss/videoDetail.scss

```diff
@@ -24,3 +24,73 @@
     }
   }
 }
+
+.videoSubtitles {
+  padding: 10px;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 100%;
+  align-items: stretch;
+
+  .videoSubtitle-card {
+    margin: 0 0 40px;
+    background-color: #ffffff;
+    width: 100%;
+
+    @include breakpoint(mobile) {
+      width: 100%;
+    }
+    @include breakpoint(phone) {
+      width: 100%;
```

Comment:
Looks like `width: 100%;` is defined for every possible viewing scenario unless I'm missing something

Reply:
OK